src/Bridges/Bridges.jl

```diff
@@ -59,6 +61,16 @@ end
 
 include("vectorizebridge.jl")
 @bridge Vectorize VectorizeBridge () (MOI.EqualTo, MOI.LessThan, MOI.GreaterThan,) () () (MOI.SingleVariable,) (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction) () ()
+include("slackbridge.jl")
+@bridge ScalarSlack ScalarSlackBridge () (MOI.Interval, MOI.LessThan, MOI.GreaterThan) () () () (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction) () ()
+@bridge(VectorSlack, VectorSlackBridge,  (), (),
+    (MOI.Nonnegatives, MOI.Nonpositives, MOI.SecondOrderCone,
+    MOI.RotatedSecondOrderCone, MOI.GeometricMeanCone,
+    MOI.PositiveSemidefiniteConeSquare, MOI.LogDetConeTriangle,
+    MOI.RootDetConeTriangle, MOI.PowerCone, MOI.DualPowerCone),
```

Comment:
The `PowerCone` should be in the next parenteses as `(MOI.PowerCone,)`

Reply:
Good catch, thanks!